parcel.go

```diff
@@ -13,48 +16,87 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 }
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+	res, err := s.db.Exec("INSERT INTO parcel (client, status, address, created_at) VALUES (?, ?, ?, ?)",
+		p.Client, p.Status, p.Address, p.CreatedAt)
+	if err != nil {
+		return 0, err
+	}
+
+	id, err := res.LastInsertId()
+	if err != nil {
+		return 0, err
+	}
+	return int(id), nil
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
-	// реализуйте чтение строки по заданному number
-	// здесь из таблицы должна вернуться только одна строка
-
-	// заполните объект Parcel данными из таблицы
-	p := Parcel{}
+	var client int
+	var status, address, createdAt string
+
+	row := s.db.QueryRow("SELECT client, status, address, created_at FROM parcel WHERE number = ?", number)
+
+	// Сканируем результат запроса в переменные
+	err := row.Scan(&client, &status, &address, &createdAt)
+	if err != nil {
+		if err == sql.ErrNoRows {
+			return Parcel{}, fmt.Errorf("посылка с номером %d не найдена", number)
+		}
+		return Parcel{}, fmt.Errorf("ошибка при получении посылки: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return Parcel{}, fmt.Errorf("ошибка при получении посылки: %w", err)
```